src/search/FindInFiles.js

```diff
@@ -96,7 +96,12 @@ define(function (require, exports, module) {
     }
 
     function nodeFileCacheComplete(event, numFiles, cacheSize) {
-        var projectName = ProjectManager.getProjectRoot().name || "noName00";
+        var projectName = ProjectManager.getProjectRoot();
```

Comment:
Small nit / bikeshedding: instead of assigning the return value `getProjectRoot()` to `projectName`, it could be a bit cleaner if it was something like:

```
var projectRoot = ProjectManager.getProjectRoot();
var projectName = projectRoot ? projectRoot.name : null;

if (!projectName) {
  projectName = "noName00";
}
```

In general I think that there should never be an situation where `projectRoot.name` is empty anyways until Brackets supports opening up with no projects at all.